plugins/inputs/docker_log/docker_logs.go

```diff
@@ -378,10 +379,10 @@ func (d *DockerLogs) Start(acc telegraf.Accumulator) error {
 		if err != nil {
 			return err
 		}
-
 		c, err = d.newClient(d.Endpoint, tlsConfig)
 	}
 	if err != nil {
+		log.Printf("%s : %s", ERR_PREFIX, err.Error())
```

Comment:
I believe this will get logged where `Start` gets called, it may not be needed.

Reply:
removed this log print